releases/vhd-notes/aks-ubuntu-1604-201903:2019.03.05.txt

```diff
@@ -0,0 +1,114 @@
+Starting build on Tue Mar 5 17:41:16 UTC 2019
```

Comment:
let's get rid of the `colon` in the filename

Reply:
underscore instead? I used the name as it appears in `az vm image list` but it's true that colon isn't great in a filename

Reply:
underscore/hyphen/period, I think anything is better than colon (or any slash character)

Reply:
went for underscore